spec/mocks/static_page_mocks.rb

```diff
@@ -0,0 +1,35 @@
+module Mocks
+  class StaticPages
+    STATIC_PAGE = {
```

Comment:
Mocks for API calls. Would appreciate any input on how to make these developer-friendly and stable.